lib/new_relic/agent/configuration/default_source.rb

```diff
@@ -1602,6 +1602,39 @@ def self.enforce_fallback(allowed_values: nil, fallback: nil)
           :allowed_from_server => false,
           :description => 'Controls auto-instrumentation of Sinatra at start up. May be one of: `auto`, `prepend`, `chain`, `disabled`.'
         },
+        :'instrumentation.stripe' => {
+          :default => 'enabled',
+          :public => true,
+          :type => String,
+          :allowed_from_server => false,
+          :description => 'Controls auto-instrumentation of Stripe at start up. May be one of: `enabled`, `disabled`.'
```

Comment:
This isn't new for Stripe, but I think it ought to be `startup`.

Reply:
I think there is a ticket out there to change all these... hoping someone picks it up for hacktoberfest! I'll make this change for Stripe though.